extensions/oidc/deployment/pom.xml

```diff
@@ -47,11 +47,27 @@
         </dependency>
         <dependency>
             <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-core</artifactId>
+            <artifactId>keycloak-admin-client</artifactId>
```

Comment:
Can you please check if Dev Services for Keycloak works with these changes ? You can do it by running for example: https://github.com/quarkusio/quarkus-quickstarts/tree/main/security-openid-connect-quickstart

and then login as `alice:alice` in Keycloak, exactly as described in: https://quarkus.io/guides/security-openid-connect-dev-services#develop-service-applications